# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/ViewportUtils.h
# Commit: 071e6527d743
# Full Hash: 071e6527d74348ac8940e60018acc76bd7b44714
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2020-05-06 09:37:16
# Regressor Bug: 1556556
# File Overlap Count: 1
# Description:
#   Bug 1556556 - Rename GetCallbackTransform() to GetVisualToLayoutTransform(). r=kats
#   
#   We will be applying and unapplying this transform in many places, and
#   thinking about those operations as "applying the callback transform" and
#   "unapplying the callback transform" is not very intuitive, especially since
# ==============================================================================

diff -r bda050135c64 -r 071e6527d743 layout/base/ViewportUtils.h
--- a/layout/base/ViewportUtils.h	Tue May 05 19:22:12 2020 +0000
+++ b/layout/base/ViewportUtils.h	Tue May 05 19:37:26 2020 +0000
@@ -12,9 +12,12 @@
 
 class ViewportUtils {
  public:
-  /* Return a "callback transform" to be applied to the coordinates of input
-     events targeting content inside the scroll frame identified by |aScrollId|.
-     The callback transform has two components:
+  /* Return a transform to be applied to the coordinates of input events
+     targeting content inside the scroll frame identified by |aScrollId|, which
+     converts from "visual coordinates" (which are the coordinates events have
+     when they arrive from APZ) to "layout coordinates" (which are the
+     coordinates used in most places by layout code). The transform has two
+     components:
      1. The pres shell resolution, representing the pinch-zoom scale
         (if the scroll frame |aScrollId| is inside the resolution, which
         is most of the time).
@@ -27,7 +30,7 @@
        up to the root, using values populated in
        APZCCallbackHelper::UpdateCallbackTransform. See that method's
        documentation for additional details. */
-  static CSSToCSSMatrix4x4 GetCallbackTransform(
+  static CSSToCSSMatrix4x4 GetVisualToLayoutTransform(
       layers::ScrollableLayerGuid::ViewID aScrollId);
 };
 
